packages/node_modules/@ciscospark/spark-core/src/spark-core.js

```diff
@@ -120,6 +122,24 @@ const SparkCore = AmpState.extend({
         });
 
       if (typeof get(attrs, 'credentials.access_token') === 'string') {
+        if (get(attrs, 'credentials.access_token').includes('Bearer') && get(attrs, 'credentials.access_token').split(' ').length - 1 === 0) {
```

Comment:
I think this ability is a perfect candidate to break out into the appropriate functions and unit test them.

Reply:
The unit test for this is not that simple.
Would have written tests for this but it's this weird reduce function.

But agree with that this should be broken out into its own function

Reply:
I think you could encapsulate the string replace logic and test different scenarios against it.

Reply:
Alright added those tests